backend/src/main/java/com/htecgroup/skynest/service/impl/FileServiceImpl.java

```diff
@@ -139,6 +133,59 @@ public List<FileResponse> getAllFilesWithParent(UUID parentFolderId) {
     return asFileResponseList(allFiles);
   }
 
+  @Override
+  @Transactional(rollbackFor = Exception.class)
+  public FileResponse updateFileContent(MultipartFile multipartFile, UUID fileId) {
```

Comment:
Because you have extracted this logic from the previous uploadFile method and extended with some additional update logic, I will try to separate that.
What I mean is, extract the mutual logic for saving the input stream into GridFs and corresponding logic for metadata in relation database. Then that method will be called in `uploadFile` and the same method plus the custom logic for update will be in other method that will be used in updateFileContent. 

Makes sense or I am missing something regarding this checks?